extension.json

```diff
@@ -110,6 +110,10 @@
               "onBeforeEventSend": {
                 "type": "string",
                 "minLength": 1
+              },
+              "onBeforeLinkClickSend": {
+                "type": "string",
+                "minLength": 1
               }
             },
             "required": ["edgeConfigId", "name"],
```

Comment:
You need to add a "function" type transform for onBeforeLinkClickSend. Look at line 129 for how it works with onBeforeEventSend.